frontend/components/group/Invitation.tsx

```diff
@@ -0,0 +1,57 @@
+import { Check, X } from "react-feather";
+
+import { Button } from "@/components/input/Button";
+import { Invitation as InvitationType } from "@/types/Invitation";
+import { declineGroupInvitation, sendGroupRequest } from "@/util/api";
+
+import { Text } from "../Text";
+
+interface InvitationProps {
+	/**
+	 * Invitation
+	 */
+	invitation: InvitationType;
+	/**
+	 * Additional classes
+	 */
+	className?: string;
```

Comment:
```suggestion
	className: string = "";
```
Otherwise "null" will be inserted into the className string if no classnames are provided